diff -ru php-4.3.6/ext/standard/info.c php-4.3.6.entropy/ext/standard/info.c
--- php-4.3.6/ext/standard/info.c	Mon Mar 15 11:39:53 2004
+++ php-4.3.6.entropy/ext/standard/info.c	Tue Apr 27 18:44:04 2004
@@ -419,6 +419,7 @@
 
 		if (!sapi_module.phpinfo_as_text) {
 			php_printf("<h1 class=\"p\">PHP Version %s</h1>\n", PHP_VERSION);
+                        php_printf("<h2 class=\"p\">www.entropy.ch Release %d (Universal Binary)</h2>\n", ENTROPY_CH_RELEASE);
 		} else {
 			php_info_print_table_row(2, "PHP Version", PHP_VERSION);
 		}
